doc/build-instructions/Build_Instructions_V8.md

```diff
@@ -405,25 +404,6 @@ tar -xzf freemarker.tgz freemarker-2.3.8/lib/freemarker.jar --strip=2
 tar --one-top-level=/cygdrive/c/temp/freetype --strip-components=1 -xzf freetype-2.5.3.tar.gz
 ```
 
-Note:
-In order to build Windows 32-bit, Please enure the freetype version supports win 32-bit application (e.g. freetype-2.4.7 which includes the "win32" folder)
-
-- To build the Freetype dynamic and static libraries, open the Visual Studio Command Prompt (VS2010) (see C:\ProgramData\Microsoft\Windows\Start Menu\Programs\Microsoft Visual Studio 2010\Visual Studio Tools) and run:
```

Comment:
We still need to build freetype libraries, but from Developer Command Prompt for VS2013 (C:\ProgramData\Microsoft\Windows\Start Menu\Programs\Visual Studio 2013\Visual Studio Tools) and running respective commands for Win 64-bit and 32-bit. Just the note for freetype to include the "win32" folder is no longer required.

Hope this helps!

Reply:
I believe that might be the case if the `configure` options are not modified.  Note that I made a change to those as well.

`--with-freetype-include=/cygdrive/c/temp/freetype/include --with-freetype-lib=/cygdrive/c/temp/freetype/lib64` becomes `--with-freetype-src=/cygdrive/c/temp/freetype`  With this in place I get the following output from configure :

```
configure: Trying to compile freetype sources with PlatformToolset=v120 to /cygdrive/c/temp/freetype/lib64/ ...
configure: Compiling freetype sources succeeded! (see freetype.log for build results)
configure: Found freetype include files at /cygdrive/c/temp/freetype/include using --with-freetype-src
configure: Rewriting POTENTIAL_FREETYPE_INCLUDE_PATH to "/cygdrive/c/temp/freetype/include"
configure: Rewriting POTENTIAL_FREETYPE_LIB_PATH to "/cygdrive/c/temp/freetype/lib64/"
checking for freetype includes... /cygdrive/c/temp/freetype/include
checking for freetype libraries... /cygdrive/c/temp/freetype/lib64/
checking if we can compile and link with freetype... yes
checking if we should bundle freetype... yes
```

Reply:
ok, that makes sense.

I just noticed a typo - adminstrative privledges.

Reply:
Updated.